src/main/java/com/playonlinux/ui/impl/javafx/installwindow/InstallWindow.java

```diff
@@ -242,10 +243,10 @@ private void setUpEvents() throws PlayOnLinuxError {
                 playOnLinuxError.printStackTrace();
             }
         });
-        searchWidget.setOnKeyPressed(event -> availableInstallerListWidget.setSearchFilter(searchWidget.getText()));
-        testingCheck.setOnAction(event -> availableInstallerListWidget.setIncludeTesting(testingCheck.isSelected()));
-        noCdNeededCheck.setOnAction(event -> availableInstallerListWidget.setIncludeNoCDNeeded(noCdNeededCheck.isSelected()));
-        commercialCheck.setOnAction(event -> availableInstallerListWidget.setIncludeCommercial(commercialCheck.isSelected()));
+        searchWidget.setOnKeyReleased(event -> availableInstallerListWidget.getFilter().setTitle(searchWidget.getText()));
```

Comment:
Using the `KeyReleased` event instead of the `KeyPressed` event fixes that the Title-Filter was hanging behind by one char.